common/src/test/org/apache/hadoop/hive/common/type/HiveDecimalTestBase.java

```diff
@@ -88,7 +86,6 @@ public BigDecimal randHiveBigDecimalNormalRange(Random r, String digitAlphabet)
     boolean negated = false;
     if (r.nextBoolean()) {
       bigInteger = bigInteger.negate();
-      negated = true;
```

Comment:
this assignment goes away; but it seems like "negated" is an unreferenced local variable....doesn't it?